components/elm/src/biogeochem/ComputeSeedMod.F90

```diff
@@ -269,8 +245,7 @@ function SpeciesTypeMultiplier(species, pft_type, component) result(multiplier)
              multiplier = 0._r8
           end if
        case default
-          write(iulog,*) subname//' ERROR: unknown component: ', component
-          call endrun(subname//': unknown component')
+          print *, ' ERROR: unknown component: ', component
```

Comment:
Please undo the deletion of all such code in this PR and wrap it with `#ifndef _OPENACC`

```
#ifndef _OPENACC
          write(iulog,*) subname//' ERROR: unknown component: ', component
          call endrun(subname//': unknown component')
#endif
```